features/step_definitions/custom_web_steps.rb

```diff
@@ -1,10 +1,6 @@
 # frozen_string_literal: true
 
-When /^I leave "([^\"]*)" blank$/ do |field|
-  fill_in field, :with => ''
-end
-
-Then /^I should see link to (.+)$/ do |page_name|
+Then "(I )(they )should see (a )(the )link to {}" do |page_name|
```

Comment:
```suggestion
Then "I/they should see a/the link to {}" do |page_name|
```

Reply:
`I/they` makes it mandatory to add one or the other, rendering the following steps invalid:
```
Then they should see a link to "A"
But should not see a link to "B"
```